python/tvm/relay/frontend/change_datatype.py

```diff
@@ -0,0 +1,78 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from ..expr import var, bind, Function
+from ..expr_functor import ExprMutator
+
+class ChangeDatatype(ExprMutator):
```

Comment:
move to relay.testing for now as it only was used for test purposes, once we reformalize, we should think about where to put it

Reply:
Sorry, this shouldn't have been in this PR. This was due to a faulty push.